doc/qbittorrent.1.md

```diff
@@ -3,34 +3,165 @@
 % January 16th 2010
```

Comment:
I would appreciate if you also update the author and date too.

Reply:
I'll update the date, but we should propbably remove the author altogether according to the recommendation in http://man7.org/linux/man-pages/man7/man-pages.7.html

Reply:
The author is currently removed. Let me know your opinion.

Reply:
I'm ok with it.